src/baltree.h

```diff
@@ -61,7 +61,7 @@ static Int height_to_size[MaxTreeDepth];
 static const Int alpha_hi = 3;
 static const Int alpha_lo = 2;
 
-static void InitBalancedTrees(void)
+static inline void InitBalancedTrees(void)
```

Comment:
is this what EXPORT_INLINE is for?

Reply:
No. That is for exporting symbols to the outside world. We do not want this here, to the contrary.